src/sys/sendfile.rs

```diff
@@ -120,6 +120,49 @@ cfg_if! {
                 )
             }
         }
+    } else if #[cfg(any(target_os = "solaris", target_os = "illumos"))] {
```

Comment:
Please rebase and use `#[cfg(solarish)]`

Reply:
yes I just saw your PR, will do.